README.md

```diff
@@ -31,6 +31,7 @@ run the following to use pctl against it:
 
 1. Create your catalog, for example there is a `examples/profile-catalog-source.yaml` file in the profiles repo
 `kubectl apply -f profiles/examples/profile-catalog-source.yaml`
-1. In a separate terminal run `kubectl -n profiles-system port-forward <profiles-controller-pod-name> 8000:8000` to enable access to the API
-1. Run `pctl --catalog-url http://localhost:8000 search <query>` to search for your profile
-1. To see more detail of a profile, run `pctl --catalog-url http://localhost:8000 show <catalog-name>/<profile-name>`
+1. Ensure the current context in kubeconfig is set to the `profiles` cluster (`kubectl config current-context` should return `kind-profiles`)
```

Comment:
is there a thing somewhere that returns a nice error if the user is pointed at a cluster which does not have the api running? (probs useful to have an error for if the api is not running in general, like they forgot to turn it on)

Reply:
that sounds reasonable, is it okay if thats in a follow up PR? I've only really touched 20% of this PR, so would like it to be merged before I fiddle around more :smile:

Reply:
yep ofc, i will create a follow up thing 👍